build/crunchy-postgres-exporter/Dockerfile

```diff
@@ -19,13 +20,22 @@ RUN if [ "$DFSET" = "centos" ] ; then \
     && ${PACKAGER} install -y \
         --setopt=skip_missing_names_on_install=False \
         postgresql${PGVERSION} \
+		gettext \
+		nss_wrapper \
     && ${PACKAGER} -y clean all ; \
 fi
 
+# needed for nss_wrapper
+RUN if [ "$BASEOS" = "rhel7" ] ; then \
+	${PACKAGER} -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm ; \
+fi
```

Comment:
I beileve we need this for `ubi7`/`ubi8`. In fact, I believe we can remove all of the RHEL bits.

Reply:
I did not have a problem installing `nss_wrapper` in `ubi7/ubi8` without installing `epel-release`.

Reply:
OK, we can remove this line then.

Reply:
These lines have been removed.